README.md

```diff
@@ -13,6 +13,7 @@ The project is primarily composed of two areas:
   - Depends on:
     - Content API
     - Data API
+    - Notifier
```

Comment:
It is required to have Notifier running if you want to test subscription emails etc. locally

Reply:
Yes, that's what I've said :smile: - see the Notifier change below